### DISCUSSION TITLE
Opening Padded Sequence from Command Line?

### DISCUSSION THREAD

**Discussion by julianjohnson2691** [ggarra13/mrv2#295] [Q&A] [answered]
Hi, rudimentary question here. I'm trying to launch mrv2 and open a 4 padded sequence from the command line in Windows. The sequence goes from 961f to 1200f. I send the first frame of the sequence as part of the command line arguments:

seq__main__v007_camera_06.0966.exr

The sequence will only start at 1001 once loaded. If I remove the 4 padding and send this:

seq__main__v007_camera_06.966.exr

It works fine. Is there any way in the windows command line arguments to let mrv2 know about the padding? Or another way perhaps using python to both launch and open the sequence with the padding? 

Thanks, Julian

**Comment by ggarra13** [comment]
HI @julianjohnson2691 mrv2 should detect the padding automatically, but we've had some problems with sequences in the past not getting detected properly.  Make sure you are using v1.2.5 of mrv2 first.

If you are, can you zip the sequence and upload it somewhere so I can get rid of the bug.  I am afraid for the time being you will have to rename the sequence.

**Reply by julianjohnson2691** [reply]
Thanks for the reply. I was using v1.1.9 so I upgraded to v1.2.5 but still getting the same problem. I'll try and zip up a portion of the sequence and send you a link as soon as I can.

**Comment by ggarra13** [comment]
These betas of v1.2.6 should fix the Windows problem you are having.  Please give them a try and report back:

.exe:
https://mega.nz/file/aORiSSoC#bToEcEgloBddM8DfMkMCl0yc6ISua2cgEOGxCrmmscc

:zip:
https://mega.nz/file/CKZ30SaB#nL8-euckZSIp-b4g4fZed4AHGhWoi0pfRZfI6GMDQdM

**Comment by julianjohnson2691** [comment]
Hi,

v1.2.6 fixes the padding problem. Thank you!

I did notice (I think) that the cmdline arguments for OCIO that I was using
in v1.9.1 don't react in quite the same way. In 1.9.1 I was just using the
-oi argument to set the working space to linear and mrv2 would
automatically default to the first 'display' (in my case sRGB). In v1.2.6
the same command line leaves the OCIO display values at None. If I try and
set either -od or -ov to 'sRGB' it still doesn't launch with the correct
display.

Julian

**Comment by ggarra13** [comment, marked as answer]
You are correct that the OCIO command-line changed.  Find a fix here:

.exe:
https://mega.nz/file/yG522TQR#Pt_FSqXm9If7lQV4TzrEEPk50M749sVqeK9CnwFKCw8

.zip:
https://mega.nz/file/3TQnlIoA#VmiQIpRfR5OC22HSyww4JwH_kpD1pL_TZ1ATbqMlYoA

**Comment by julianjohnson2691** [comment]
Thanks very much for those fixes. It works perfectly now!
